modules/core/src/lib/deck.ts

```diff
@@ -988,7 +992,7 @@ export default class Deck {
       deck: this,
       parentElement: this.canvas?.parentElement
     });
-    this.widgetManager.add(new Tooltip(), {placement: 'fill'});
+    this.widgetManager.addDefault(new Tooltip());
```

Comment:
Nit: Is there any way to avoid `addDefault()`? Such "global" registration is usually best avoided if possible. Can the tooltip be added when it is actually used?

Reply:
We'd want to consider if changing it could break something since the 8.9 and prior always adds a tooltip to the container.